image-overview-templates/new-names/device-bacnet-arm64.md

```diff
@@ -4,11 +4,6 @@ $$./common/quick-reference.md
 
 # Supported tags and respective Dockerfile links
 
-$$./common/version-jakarta-lts.md |
-        - [Dockerfile](https://github.com/edgexfoundry/device-bacnet-c/blob/v2.1.0/scripts/Dockerfile.alpine)
-$$./common/version-ireland.md |
-        - [Dockerfile](https://github.com/edgexfoundry/device-bacnet-c/blob/v2.0.0/scripts/Dockerfile.alpine)
-
```

Comment:
Deleted these since the only tag is the new 2.3.0, which I assume the automation will add here, correct?

Reply:
No we need to manually add the version here.